diff --git a/node_modules/jest-resolve/build/isBuiltinModule.js b/node_modules/jest-resolve/build/isBuiltinModule.js
index 8569160..289f474 100644
--- a/node_modules/jest-resolve/build/isBuiltinModule.js
+++ b/node_modules/jest-resolve/build/isBuiltinModule.js
@@ -35,5 +35,6 @@ const BUILTIN_MODULES = new Set(
 );
 
 function isBuiltinModule(module) {
-  return BUILTIN_MODULES.has(module);
+  const bareName = module.startsWith('node:') ? module.slice(5) : module;
+  return BUILTIN_MODULES.has(bareName);
 }
